nuxeo-mongodb-audit/src/main/resources/OSGI-INF/mongodb-audit-contrib.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.mongodb.audit.contrib.documentation">
+
+  <documentation>
+    MongoDB Audit backend uses MongoDBComponent. By default, if no MongoDB connection with id 'audit' exists, we will
+    use the 'default' one.
+
+    You could override the collection name used for audit by contributing to ConfigurationService like this:
+    <code>
+      <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+        <property name="nuxeo.mongodb.audit.collection.name">myCollection</property>
+      </extension>
+    </code>
+    By default, backend will use 'audit' as collection name.
+  </documentation>
+
+  <require>org.nuxeo.mongodb.core.MongoDBComponent</require>
+
+</component>
```

Comment:
Because MongoDB extension can be installed to have directories backed by MongoDB. Then when we will merge this project into Nuxeo Platform we need something to activate either Elasticsearch or MongoDB as the default Audit backend.

So we use template to enable audit and/or directory features.